napalm_ios/ios.py

```diff
@@ -20,6 +20,8 @@
 import os
 import uuid
 import tempfile
+import copy
+import ipaddress
```

Comment:
It is mostly because we already started using netaddr in this library and it would be quite a bit of work to change (and I don't to be using two different libraries for basically the same purpose). But you are right the helper in base returns a string...

So I think we will need to write a function in napalm-ios that checks if it is V4 or V6 using netaddr. I can then migrate this to napalm-base (I don't want you to be gated by that since that would take a while to get done).